### PR TITLE
docs(drawer): Add docs on the site

### PR DESCRIPTION
**Pull request by matheusps**
## Summary

Add docs on the site.

### CODE REVIEW

File: packages/docs/pages/components/drawer.mdx [hunk 1]
```diff
@@ -1,20 +1,82 @@
...
+| [DrawerTrigger](/components/drawer/drawer-trigger) | A composition-first component that toggles the [DrawerPopover](/components/drawer/drawer-popover) |
+| [DrawerPopover](/components/drawer/drawer-popover) | Popover box of the Drawer |
+| [DrawerHeader](/components/drawer/drawer-header) | The fixed Header |
+| [DrawerHeading](/components/drawer/drawer-heading) | The title of the Drawer |
+| [DrawerDismiss](/components/drawer/drawer-dismiss) | Dismisses the [DrawerPopover](/components/drawer/drawer-popover) |
+| [DrawerContent](/components/drawer/drawer-content) | Main content inside of the [DrawerPopover](/components/drawer/drawer-popover) |
+| [DrawerFooter](/components/drawer/drawer-footer) | The fixed Footer mainly used for actions |
+
+
+The example code below, illustrates how this components interact with each other:
```
Comment (vaporwavie) [inline packages/docs/pages/components/drawer.mdx:25]: **typo**: ~~this~~ for _these_ instead

File: packages/docs/pages/components/drawer/drawer-footer.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# DrawerFooter
+
+<ComponentDescription name="drawer-footer" />
+
+<Preview name="drawer" />
+
+## Optional props
+
+All props of `<footer />` Jsx element.
```
Comment (vaporwavie) [inline packages/docs/pages/components/drawer/drawer-footer.mdx:9]: nit(optional): prefer JSX over Jsx

**Review by vaporwavie — COMMENTED** [review]
looks good! left my two cents (literally) but the examples are great.

**Review by lucasaarcoverde — APPROVED** [review]
LGTM, just one thing. When I open the drawer it happens an layout shifts on the page, is it just for me? Maybe something related with the doc site itself

**Pull request merged**